src/components/charts/StocksChart.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC } from "react";
+import { Line } from "react-chartjs-2";
+
+type StocksChartScreenType = {
+  stocksData?: any;
+  priceData?: any;
+};
+
+const StocksChart: FC<StocksChartScreenType> = ({
+  stocksData,
+}): JSX.Element => {
+  const canvas = document.createElement("canvas");
```

Comment:
You don't need to create any canvas, chart library will create one for the component